setup.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from setuptools import Command, setup
+from setuptools import find_packages
+from unittest import TestLoader, TextTestRunner
+
+from os import environ, path
+
+PROJECT = 'oca-decorators'
```

Comment:
I'd love it if we change the name of this, now that it's not yet too late. Calling it `oca-decorators` seems to be too narrowing for possible helpers that are not decorators (context managers, test classes, data loaders, mock helpers, etc.).

I'd just call this `ocalib` (or `ocahelpers`, `odoohelpers`, `oca` or similar). The code would result into:

```
from ocalib import decorators
...
@decorators.foreach()
def what(self):
    ...
```

We could even decide to provide decorators in the main namespace, although this is not so important:

```
import ocalib
@ocalib.foreach()
def lalala()...
```

Reply:
+1 for wide name than only decorator

Reply:
I'm indifferent on the naming. It was done this way so that we could make micro libraries and avoid a monolithic package. The conversation started [here](https://github.com/OCA/server-tools/issues/944#issuecomment-336176347). CC @lmignon